content/cpp/concepts/math-functions/terms/trunc/trunc.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'trunc()' 
+Description: 'Returns the integral part of the argument.' 
+Subjects:
+  - 'Computer Science'
+Tags: 
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`trunc()`** function rounds the specified argument towards zero and returns the nearest integral value not larger in magnitude than the argument. The [data type](https://www.codecademy.com/resources/docs/cpp/data-types) of the return value can be `double`, `float`, or `long double`.
+
+## Syntax
+
+```pseudo
+std::trunc(x);
+```
+
+The `x` parameter is the value to be truncated.
```

Comment:
```suggestion
The `x` parameter is the value to be truncated.

The [data type](https://www.codecademy.com/resources/docs/cpp/data-types) of the return value can be `double`, `float`, or `long double`.
```

Reply:
@sudojarvis Looks like this was resolved without the change being implemented. We should include this text as I think its important to describe the return type(s) in the "Syntax" section.